### DISCUSSION TITLE
2024: News about A.I, Machine Learning and Robots (high-value and specific to our group)

### DISCUSSION THREAD

**Discussion by geekscape** [ai-ml-robots/ai-ml-robots.github.io#2] [General]
**2024-08-26: [Beijing robot conference has 27 humanoid robots (a record)](https://www.cnbc.com/2024/08/26/teslas-optimus-faces-humanoid-competition-at-beijing-robot-conference.html)**

_Press the **[Newest]** button to get the most recent news first_

**Comment by geekscape** [comment]
**2024-09-11: AI in Manufacturing speaker and networking event** ... at Monash, Clayton from 4:30 pm to 6:30 pm
- https://amtil.com.au/events/amtil/ai-in-manufacturing-speaker-and-networking-event
- Tickets are AUS$33 for members of the public (non-members of AMTIL)

Details ...
- Q & A with the expert panel
- Tour the [Monash Makerspace](https://www.monash.edu/engineering/future-students/studying-engineering-at-monash/facilities/monash-makerspace) and [Monash Deep Neuron team](https://www.deepneuron.org/) display
- Tour Monash Smart Manufacturing and see the [Monash Automation team](https://monashautomation.com/) in action
- Canapes and Networking
- Beer tasting with the [Monash Brew Lab Team](https://www.monashbrewlab.com/)

**Comment by geekscape** [comment]
2024-09-17: Festival of AI and Robotics (FAIR) ... in Melbourne on 17th and 18th September 2024 ... NOW POSTPONED !**

[LinkedIn post](https://www.linkedin.com/posts/the-fair-festival-of-ai-robotics_techexplained-australiatech-techtoday-activity-7230827804529934337-59vN) and [web-site](https://thefair.technology)

This large multi-day event with many speakers and involvement from government and industry ... has been postponed with less than two weeks to go.  No reason has been given for the postponement.

It looks like this was much more of a business event ... targeting Small to Medium business Enterprises (SMEs) ... rather than a technical conference.  Even so, it is good to see interest in A.I/ML _**and**_ robots ... and quite curious as to why it isn't going ahead :thinking:

**Comment by ai-ml-robots** [comment]
**2024-09-17: ML/A.I Melbourne meet-up: Extremely Constrained Machine Learning on Edge Devices** by Mat Kelcey

- https://www.meetup.com/machine-learning-ai-meetup/events/301702113

Mat is a very experienced ML developer and over recent years has been focusing on ML deployed on Edge devices, e.g [microcontrollers](https://en.wikipedia.org/wiki/Microcontroller) and [FPGAs](https://en.wikipedia.org/wiki/Field-programmable_gate_array).  His talks are always detailed and excellent.  Highly recommended.

This talk will be relevant for using ML on very small sensors and microcontrollers, like we'd tend to use in robotics (and other applications).

**Comment by geekscape** [comment]
**2024-10-30: [Boston Dynamics' Atlas is autonomously moving engine covers](https://www.youtube.com/watch?v=F_7IPm7f1vI)**

**2024-10-31: [Meta: Advancing embodied AI through progress in touch perception, dexterity and human-robot interaction](https://ai.meta.com/blog/fair-robotics-open-source)**

**2024-10-31: [Physical Intelligence (π) - π0: Our First Generalist Policy](https://www.physicalintelligence.company/blog/pi0)**

**2024-11-04: [Meta’s former hardware lead for Orion is joining OpenAI](https://techcrunch.com/2024/11/04/metas-former-hardware-lead-for-orion-is-joining-openai)**

**2024-11-06: [NVIDIA Advances Robot Learning and Humanoid Development With New AI and Simulation Tools](https://blogs.nvidia.com/blog/robot-learning-humanoid-development)**
